Makefile

```diff
@@ -178,3 +178,22 @@ build-webhook:
 .PHONY: docker-build-webhook
 docker-build-webhook:
 	docker build -f cmd/admission-webhook-server/Dockerfile -t $(ADMISSION_WEBHOOK_SERVER) .
+
+.PHONY: check-working-tree-clean
+check-working-tree-clean:
+	@test -z "$$(git status --porcelain)" || (echo "Commit all changes before running this target"; exit 1)
+
+.PHONY: check-update-assets
+check-update-assets: check-working-tree-clean update-assets
+	@test -z "$$(git status --porcelain)" || (echo "Please run make update-assets and commit the changes."; git status --porcelain; exit 1)
```

Comment:
Previously we used `git diff` when checking for assets. I think we should do that here too because asset changes are usually not very large.

Reply:
Only for assets right?

Reply:
Yes, assets should report `git --no-pager diff` to the user.